Exercise_D and E/assets/style/index.css

```diff
@@ -2,3 +2,58 @@
 This css sheet is linked to practice.html
 You can style tags, classes (.) or ids (#) from this file.
 */
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+html {
+    font-family: 'Roboto', sans-serif;
+}
+.header{
+background-color: white;
+}
+.mid{
+    text-align: center;
+    background-image: linear-gradient(rgba(0,0,0,0.7), rgba(0,0,0,0.7)), url(../images/flowershop.jpg) ;
+    background-position: center;
+    background-size: cover;
+    height:700px;
+}
+.mid > p{
+    color: white;
+    font-size: 50px;
+    padding-top: 300px;
+    margin-bottom: 50px;
+}
+.mid > a{
```

Comment:
If you want to remove the blue line on hover, you can add

text-decoration: none

to the element when its hovered.

try something like:
.mid > a :hover {
color: white;
text-decoration: none;
}